.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
 -   repo: https://github.com/ambv/black
-    rev: 19.3b0
+    rev: 19.10b0
```

Comment:
Changed this so that it is compatible with the version in `requirements_dev.txt`